README_DEVELOPERS.md

```diff
@@ -68,6 +73,26 @@ drone exec --matrix TARGET=test
 
 ### In a Cloud CI
 
-We provide configuration YAML files for [Travis CI](https://travis-ci.org/) ([.travis.yml](.travis.yml)) and [Shippable](https://app.shippable.com/) ([shippable.yml](shippable.yml)). Enable one of them for your forked repo (we recommend Travis). 
+We provide configuration YAML files for [Travis CI](https://travis-ci.org/) ([.travis.yml](.travis.yml)) and [Shippable](https://app.shippable.com/) ([shippable.yml](shippable.yml)). Enable one of them for your forked repo (we recommend Travis).
 
 Please make sure that each pull request passes all automatic checks before submitting it.
+
+## Commit Message Guidelines
+
+We have a few rules of formatting commit messages and pull request titles:
+
+- Use the imperative mood in the subject line
+- Capitalize the subject line
+- Do not end the subject line with a period
+- Limit the subject line to 50 characters
+- Separate the subject from the body with a blank line
+- Wrap the body at 72 characters
```

Comment:
Do we really need this rule about 72 chars wrapping?
Anybody works in an old terminal or reviews patches in emails?

Reply:
@AlekseyMartynov We can discuss it in our chat, in the corresponding thread. In this PR I'm just adding the rules, which we are following now.

Reply:
It's not exactly true.
Looks more like a a sloppy copy-paste from https://chris.beams.io/posts/git-commit/

Refering to that source looks more efficient to me.

Reply:
I can add the reference to the source. It's a good idea.